src/components/SharedComponents/UploadStatus/UploadStatus.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+
+import INatIcon from "components/INatIcon";
+import { View } from "components/styledComponents";
+import * as React from "react";
+import CircularProgressBase from "react-native-circular-progress-indicator";
+import colors from "styles/tailwindColors";
+
+type Props = {
+  color: string,
+  completeColor: string,
+  progress: number
+}
+
+const UploadStatus = ( { color, completeColor, progress }: Props ): React.Node => (
+  <View className="relative justify-center items-center">
+    {( progress < 0.05 )
+      ? <INatIcon className="m-1" name="status-saved" color={colors.darkGray} size={50} />
+      : (
+        <>
+          {( progress < 1 )
+            ? <INatIcon className="absolute" name="upload-arrow" color={color} size={25} />
```

Comment:
Can you change the icons so that the size of the circle is 33px by default. The design specs have 31x31 when using these status icons in the observation list screen. (The save icon is acually 33x33 in the specs, but we would need to check if the layout jumps when the progress is moving.)